services/web-development.html

```diff
@@ -0,0 +1,191 @@
+---
+permalink: /services/web-development
+layout: default
+title: From MVP to Сomplete Product. Web Development Services | Active Bridge
+description: Enhance your business processes leveraging our web application
+             development services. We excel at crafting digital experiences that
+             both look and work impressive.
+---
+
+<div class="container web_dev_services development">
+  <div class="web_dev_services__main wrap">
+    <div class="row">
+      <div class="header-2">{{ site.data.web_dev_services.title }}</div>
+      <div class="description">{{ site.data.web_dev_services.description }}</div>
+      <div class="web_dev_services__list">
+        <div class="header-3">{{ site.data.web_dev_services.expertise.title }}</div>
```

Comment:
it's missing some gap between image and text:
![image](https://github.com/katatsu12/live_blog/assets/22080708/5c69e832-d965-4b4e-a190-7e3a7398ca37)

Reply:
same here:
![image](https://github.com/katatsu12/live_blog/assets/22080708/040c49ec-9ae7-4c4f-8622-03ec42bdef5f)